src/lib.rs

```diff
@@ -1167,6 +1167,20 @@ impl Log for NopLogger {
     fn flush(&self) {}
 }
 
+#[cfg(feature = "std")]
+impl Log for std::boxed::Box<dyn Log> {
```

Comment:
This can be made more general to `impl<T> Log for Box<T> where T: ?Sized + Log { ... }`